presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -27,15 +27,15 @@ Other HTTP endpoints include:
 * GET: v1/info
 * GET: v1/status
 
-The request/response flow of Prestissimo is identical to Java workers. The
+The request/response flow of C++ based Presto is identical to Java workers. The
 tasks or new splits are registered via `TaskUpdateRequest`. Resource
 utilization and query progress are sent to the coordinator via task endpoints.
 
 
 Remote Function Execution
 -------------------------
 
-Prestissimo supports remote execution of scalar functions. This feature is
+C++ based Presto supports remote execution of scalar functions. This feature is
 useful for cases when the function code is not written in C++, or if for
```

Comment:
reasons,

Reply:
@elharo, I don't understand what you mean with this comment. Please clarify.

Reply:
@elharo, I don't understand what you mean with your comment " reasons, ". Please clarify.

Reply:
add comma after "reasons"
possibly add comma after "or" too

Reply:
Thanks! I was looking at line 39 where you added this comment on and didn't see "reasons" in the line you appeared to be commenting on, not realizing that you were commenting about line 40 instead. 

I'll fix this as follows, in this screenshot of the comment and suggestion feature in GitHub.
<img width="841" alt="Screenshot 2024-05-14 at 10 02 18 AM" src="https://github.com/prestodb/presto/assets/7013443/a7b25a07-b69f-4049-8705-6544950823f2">